services/configuration/src/schema.rs

```diff
@@ -49,44 +52,54 @@ encoding_struct! {
 }
 
 lazy_static! {
-    static ref NO_VOTE_BYTES: Vec<u8> =
-        Vote::new_with_signature(&PublicKey::zero(), &Hash::zero(), &Signature::zero(),)
-            .into_bytes();
+    static ref NO_VOTE_BYTES: Vec<u8> = vec![0u8];
 }
 
 /// A enum used to represent different kinds of vote, `Vote` and `VoteAgainst` transactions.
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-#[serde(tag = "vote_for", rename_all = "lowercase")]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Copy)]
+#[serde(rename_all = "lowercase", tag = "vote_type", content = "tx_hash")]
 pub enum VotingDecision {
-    /// `Vote` transaction variant.
-    Yea(Vote),
-    /// `VoteAgainst` transaction variant.
-    Nay(VoteAgainst),
+    /// `Vote` transaction `Hash` variant.
+    Yea(Hash),
+    /// `VoteAgainst` transaction `Hash` variant.
+    Nay(Hash),
```

Comment:
Be sure to note this in the changelog (at some point) as such change will require restructuring light clients a bit if they relied on vote transactions being present in the voting decision directly.